services/distribution/src/test/resources/dgc/country-list.json

```diff
@@ -0,0 +1,29 @@
+[
```

Comment:
Why is this list added twice (the other time at services/distribution/src/main/resources/dgc/country-list.json)?